HW8.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <meta charset="UTF-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+   </head>
+   <body>
+      <body style=" margin: auto; padding: 10px; color: red; border: 1px solid black; border-radius: 5px; width: 95%; " >
+         <h1>основи роботи з DOM</h1>
+         <section class="firstSection">
+            <h2>DOM дерево</h2>
+            <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. </p>
+         </section>
+         <div>
+            <h2 id="headerTwo">навігація по DOM дереву</h2>
+            <p class="text"> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. </p>
+            <h3>список</h3>
+            <ul>
+               <li class="text">пункт 1</li>
+               <li>пункт 2</li>
+               <li>пункт 3</li>
+               <li class="text">пункт 4</li>
+               <li>пункт 5</li>
+               <li class="text">пункт 6</li>
+            </ul>
+            <button class="redBtn">оновити рівень</button>
+            <div class="hatredLevelBlock">
+               <label class="hatredLevelLabel">поточний рівень:</label>
+               <span class="hatredLevelCounter"></span>
+            </div>
+         </div>
+      </body>
+      <script>
+         'use strict'
+         const _1 = document.getElementById('headerTwo'),
+            _2 = document.querySelectorAll('section h2')[0],
```

Comment:
```suggestion
           **тут має бути ключове слово const** **тут має бути нормальна назва змінної** = document.querySelectorAll('section h2')[0] тут має бути **;**
```
І так всюди і завжди.